src/Scope.spec.ts

```diff
@@ -488,12 +488,12 @@ describe('Scope', () => {
             const validateStartScope = sinon.spy();
             const validateEndScope = sinon.spy();
             await program.addOrReplaceFile('source/file.brs', ``);
-            await program.addOrReplaceFile('components/comp.xml', `
-                <?xml version="1.0" encoding="utf-8" ?>
+            await program.addOrReplaceFile('components/comp.xml',
```

Comment:
If you didn't like having to move all opening backticks to the next line (and then trailing backticks as well), you could use the `trim` template function. [Here's](https://github.com/rokucommunity/brighterscript/blob/abae186cc0857eb866d54f05f5a7a0b86cf41197/src/testHelpers.spec.ts#L35) its definition, [here's](https://github.com/rokucommunity/brighterscript/blob/abae186cc0857eb866d54f05f5a7a0b86cf41197/src/files/BrsFile.spec.ts#L2235) how it's used. I think that would help reduce the churn on a bunch of these tests.

Reply:
nice, that helped a lot. Thanks!